src/bin/lpython.cpp

```diff
@@ -29,6 +29,7 @@
 #include <libasr/config.h>
 #include <libasr/string_utils.h>
 #include <libasr/lsp_interface.h>
+#include <lpython/python_kernel.h>
```

Comment:
You can probably just put this behind `#ifdef HAVE_LFORTRAN_XEUS` and that should do it for now. We can later refactor it differently.